components/bl-weather-component/styles/index.less

```diff
@@ -0,0 +1,9 @@
+@bl-customComponent-weather-size: 100%;
+
+.bl-customComponent-weather {
```

Comment:
`bl-customComponent-openWeather` or change root class to` bl-customComponent-weather` in jsx.